pkgs/applications/blockchains/stellar-core/default.nix

```
@@ -1,22 +1,24 @@
 { lib, stdenv, fetchFromGitHub, autoconf, libtool, automake, pkg-config, git
-, bison, flex, postgresql, ripgrep }:
+, bison, flex, postgresql, ripgrep, libunwind }:
 
 stdenv.mkDerivation rec {
   pname = "stellar-core";
-  version = "17.0.0";
+  version = "18.5.0";
 
   src = fetchFromGitHub {
     owner = "stellar";
     repo = pname;
     rev = "v${version}";
-    sha256 = "1ngl8yjqb8xzhdwzlxzzxf14q2hgwy2ysb17sn5380rrn0jswin1";
+    sha256 = "sha256-wEi22R4zb8d5CJV5eWb776Yob8B6Ok4FrbYI0SGM0H8=";
     fetchSubmodules = true;
   };
 
-  nativeBuildInputs = [ automake autoconf git libtool pkg-config ripgrep ];
+  nativeBuildInputs = [ automake autoconf git libtool pkg-config ripgrep libunwind ];
```

this should preferably be

```suggestion
  nativeBuildInputs = [ autoreconfHook git pkg-config ripgrep libunwind ];
```

https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/stdenv.chapter.md#autoconf-setup-hook-autoconf

---

```suggestion
  nativeBuildInputs = [ automake autoconf git libtool pkg-config ripgrep ];
```

`libunwind` is a library so it should be in `buildInputs`

---

Switching to autoreconfHook is causing build failures I'm currently investigating. I've updated the file with the libunwind changes though.